flint/interface.py

```diff
@@ -33,6 +33,10 @@ def html_to_rdl(html: str) -> str:
         result = result.replace(html_tag, rdl_tag)
     return result
 
+@cached
+def get_infocardmap() -> dict:
+    """Return a dict of each ID in infocardmap.ini mapped to the other ID"""
+    return {id0: id1 for id0, id1 in ini.parse(paths.construct_path("DATA/INTERFACE/infocardmap.ini"))[0][1]["map"]}
```

Comment:
```suggestion
    return dict(ini.parse(paths.construct_path('DATA/INTERFACE/infocardmap.ini'))[0][1]['map'])
```
But this chain of lookups seems rather fragile to me, not sure if it can be improved though

Reply:
Not a huge fan of this way of doing it either, but I don't know of a more elegant way